pkgs/top-level/qt6-packages.nix

```diff
@@ -90,6 +90,8 @@ makeScopeWithSplicing' {
 
   waylib = callPackage ../development/libraries/waylib { };
 
+  wayqt = callPackage ../development/libraries/wayqt { };
```

Comment:
Only noticed this now but, following RFC140, new packages should go to by-name:

https://github.com/NixOS/rfcs/pull/140

Other than that, LGTM

Reply:
This RFC only addresses the top-level attribute namespace, but this is a qt library, we need it to be built for every qt version in `qt6-packages.nix`

Reply:
Ah indeed, this isn't all-packages. My bad!